.github/workflows/ci.yml

```diff
@@ -44,15 +43,8 @@ jobs:
             ${{ runner.os }}-test-
             ${{ runner.os }}-
       - uses: julia-actions/julia-buildpkg@v1
-# The following is needed for Julia <=0.8.4 on Linux OS 
-# due to old version of libstcxx used by Julia
-      - name: "Export LD_LIBRARY_PATH envrioment variable"
-        if: ${{matrix.version == '1.6'}}
-        run: echo "LD_LIBRARY_PATH=/home/runner/.julia/conda/3/x86_64/lib" >> $GITHUB_ENV
       - uses: julia-actions/julia-runtest@v1
-        env:
-          LD_LIBRARY_PATH: /home/runner/.julia/conda/3/x86_64/lib
       - uses: julia-actions/julia-processcoverage@v1
```

Comment:
Do we know if something like this is still needed. It would seem to me that this issue is orthogonal to the PythonCall/PyCall choice. 

@OkonSamuel What say you?

Reply:
This hack was due to outdated `libstdcxx` libraries in julia. `PythonCall.jl` handles this now, and restricts python package versions to only those that are compatible with the current julia session.